azurepipelines/e2e_test/scenarios/ubuntu-18.04-amd64/self_upgrade.py

```diff
@@ -65,7 +65,7 @@ def test_AptDeployment(self):
         #
         connectionStatus = ""
         for i in range(0,test_connection_timeout_tries):
-            connectionStatus = self.duTestHelper.GetConnectionStatusForDevice(test_device_id)
+            connectionStatus = self.duTestHelper.GetConnectionStatusForModule(test_device_id,"IoTHubDeviceUpdate")
```

Comment:
please run all of our python scripts through autopep.

```
pip install autopep8
autopep8 -i script.py
```

bonus points for adding a new script to scripts that does the formatting like i created for cmake and clang format.

bonus bonus points for adding to the git precommit script we have.

Reply:
Will address in separate PR. 

@shiyi-peng good to have this in general for the rework too.